rules/windows/registry_event/sysmon_runonce_persistence.yml

```diff
@@ -0,0 +1,25 @@
+title: Run Once task configuration in the registry
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh '@v3t0_'
+status: experimental
+date: 2020/10/15
+references:
+    - https://twitter.com/pabraeken/status/990717080805789697
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Runonce.yml
+tags:
+    - attack.defense_evasion
+    - attack.t1112
+logsource:
+    product: windows
+    service: sysmon
```

Comment:
I believe you can also change it to the `registry_event ` here (:

Reply:
Thank you for the review. :) 
I've changed it as per your suggestions. I can't rebase as per the instructions. It says that **The previous cherry-pick is now empty, possibly due to conflict resolution.**
If I do a git commit to the same branch, would it be okay?

Reply:
@yugoslavskiy